src/dom-to-image-more.js

```diff
@@ -298,7 +313,8 @@
         }
 
         function processClone(original, clone) {
-            if (!util.isElement(clone)) {
+            if (!util.isElement(clone)) { return Promise.resolve(clone); }
+            if (!util.isElement(clone) || util.isShadowSlotElement(original)) {
```

Comment:
Not sure how _both_ of these lines are supposed to be here... might be a merge issue.

Reply:
Resolved in 3c6b9c7